app/src/main/kotlin/com/wire/android/workmanager/worker/MigrationWorker.kt

```diff
@@ -104,11 +109,11 @@ suspend fun WorkManager.enqueueMigrationWorker(): Flow<MigrationData> {
     val request = OneTimeWorkRequestBuilder<MigrationWorker>()
         .setExpedited(OutOfQuotaPolicy.RUN_AS_NON_EXPEDITED_WORK_REQUEST)
         .setBackoffCriteria(BackoffPolicy.EXPONENTIAL, MIN_BACKOFF_MILLIS, TimeUnit.MILLISECONDS)
-        .setConstraints(
-            Constraints.Builder()
-                .setRequiredNetworkType(NetworkType.CONNECTED)
-                .build()
-        )
+//        .setConstraints(
+//            Constraints.Builder()
+//                .setRequiredNetworkType(NetworkType.CONNECTED)
+//                .build()
+//        )
```

Comment:
If you're sure about that then let's just remove it and not leave commented code

Reply:
i was testing something with this one, reverted